packages/arb-token-bridge-ui/src/components/common/Footer.tsx

```diff
@@ -37,7 +37,7 @@ const footerLinks: FooterLink[] = [
 
 export function Footer() {
   return (
-    <footer className="z-10 flex justify-center">
+    <footer className="z-[1] flex justify-center">
```

Comment:
Reason - when explore panel is not there, our footer's z-index conflicted with the `manage token lists` button and didn't allow the e2e test to click on it. Fixed it by keeping z-index slightly lower.

![image](https://user-images.githubusercontent.com/7558499/231450124-bda7861b-e622-45b4-8920-ce589ab2717d.png)